cmd/daemon/apis/apis.go

```diff
@@ -0,0 +1,60 @@
+package apis
+
+import (
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/lunarway/release-manager/internal/log"
+	"go.opencensus.io/exporter/jaeger"
+	"go.opencensus.io/plugin/ochttp"
+	"go.opencensus.io/trace"
+)
+
+// All of the configuration necessary to run a fluxcloud API
+type APIConfig struct {
```

Comment:
Maybe this is really just an `API` ?

If the package is named `flux` that becomes `api := flux.API{}` with which we can call `flux.HandleV6(api)` etc.

Reply:
Makes sense. Didn't go into it, as it is called what it is called in fluxcloud